### PR TITLE
Pending help panel work

### PR DESCRIPTION
**Pull request by reginapramesti**
### What this PR does

Fixes https://github.com/TerriaJS/nsw-digital-twin/issues/244

I've essentially tried to accommodate all of the requests that @vicborgy has made that was still pending from when I first implemented it during the launch. I've tried to tick off all the boxes that he's listed on the issue.

### Checklist

-   [x] No unit tests: These changes are more visual and is probably better for @vicborgy to have a look and test himself rather than relying on unit tests (though happy to be told otherwise). Also I don't believe unit tests exist for the help UI 😬
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by soyarsauce** [comment]
Hi @reginapramesti - is this still WIP or good to look at?

**Comment by reginapramesti** [comment]
I'm done working on it, but I'm still waiting on Vic's feedback on how things look visually, so I suppose still WIP for now

**Comment by soyarsauce** [comment]
Hi @reginapramesti - https://github.com/TerriaJS/nsw-digital-twin/issues/244#issuecomment-603641784 based on this comment does this mean it's ready? 😀

**Comment by soyarsauce** [comment]
image

Text is defined as 400 but loading in as 600, have updated with 
https://github.com/TerriaJS/terriajs/pull/4141/commits/f8cad0635325c9b30550b7065aa81da987146985

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Have a look at how I'm thinking about dealing with this (both the width and fill requiring custom css everywhere) and let me know what you think?:
https://github.com/TerriaJS/terriajs/pull/4162/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=.tsx#diff-2975e3acc006dfdbf49352a33d1e25f3R84

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: This is probably better dealt with using something like `<Spacing right={5} />` & `<Spacing bottom={17} />` (factor of 5 across our spacing/padding components) so that we can scale it accordingly for when we add in the mobile ui to this system, but I'm happy to leave this as is for now - what do you think?

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: This is better done as a prop called `extraBold` in similar vein to bold/semiBold etc

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Use the colors provided in theme
(either as a new prop in Text.jsx, or passing in something like
https://github.com/TerriaJS/terriajs/pull/4162/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=.tsx#diff-c671962cbc92db588e71d07568708422R128
and
https://github.com/TerriaJS/terriajs/pull/4162/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.scss&file-filters%5B%5D=.svg&file-filters%5B%5D=.tsx#diff-2975e3acc006dfdbf49352a33d1e25f3R98

to avoid hardcoded colors), and we can raise the issue of making color palettes more consistent across each app.

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: Similar to StyledIcon.jsx mention above

File: lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx]: Propify this combination of font-size & line-height (looks like it sits between the existing props `large` & `heading`)

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx
Comment (soyarsauce) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: (You can still do this extra stuff like you have, it would just be something like.. `styled(StyledIcon)`)

File: lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx]: Another hardcoded color

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Another hardcoded color

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: The default is
```
font-size: 13px;
line-height: 20px;
```

Now that Vic's asked to bump it up to 14px, medium + 18 becomes
```
font-size: 14px;
line-height: 18px;
```

Let's remove this and let it flow back to 20px (in the design the RHS stuff is 20px too)

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: Another hardcoded, never seen before color (I know it's from the design and you are just following that, but we need to raise it back to Vic when these things come up - in the absence of a palette of defined colours in the designs, we need to draw from existing colors/color-variable-names, and if there are new ones that should be added, we update current variables in a particular app to use those).

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: I think this is a copy-paste-but-didn't-rename-name, little confusing to see compassicon/compasswrapper when it's got nothing to do with the compass 😆

**Review by soyarsauce — CHANGES_REQUESTED** [review]
🎉 👏  Really really awesome work @reginapramesti - just a few questions. So please write back what you think on the Icon strategy that we should slowly adopt - if you think it's a good approach, happy for you to resolve those comments, then raise another issue to fix those icons in another issue/PR once `mobx-new-searchui` is merged if that helps.

The hardcoded comments are just to make sure they are all covered 🙂

File: lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx
Comment (soyarsauce) [inline lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx]: Another hardcoded color

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: Yes! My bad whoops 😆

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Yup will start doing that from now on

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Definitely a cleaner solution, will start doing it this way 😄

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: From what I understand, Spacing right/bottom just adds padding to the bottom and right side of the component right? Are you saying you might modify the spacing class to include bottom+top and right+left?

Comment (soyarsauce) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: If you put spacing with `right` on the left of something it will achieve the same thing, think of it as "x / y axis" props instead of right/down?

Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Ohhh right yes that makes sense!

Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx]: Slack discussion: 
> maybe you can tackle it another way, continue to use padding but load in the whole idea of "ratio(5px) * factor(17 or whatever)" through the themes..

> in fact you can already do that with box

> see `paddedHorizontally` && `paddedVertically`

& wait for Wing's branch to be merged to mobx and I'll proceed with this

File: lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx]: Made it a `subHeading` for now, since it's quite a fair bit bigger than `extraLarge` (what I would've otherwise named it). 

Are you still planning on bringing the h1-h6 headings, or something along those lines, from terrace that you mentioned a while back? 🤔

File: lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx]: Replaced this with `this.props.theme.textDarker` 😄

**Comment by reginapramesti** [comment]
- [x] Vic wants the text colour for help menu titles and paragraphs to `$dark-lighter: #595b60;`

**Comment by reginapramesti** [comment]
Finally finished addressing all of the comments ✔️ I think most things look better!

P.S. @soyarsauce I updated `ViewState.ts` so if you hide all of .ts files when reviewing, you may want to specifically look for `ViewState.ts` :)

**Comment by soyarsauce** [comment]
Hi @reginapramesti - the scss types have regressed but I know what needs to be done to fix them, will let you know

**Comment by soyarsauce** [comment]
@reginapramesti use the change in here: https://github.com/TerriaJS/nsw-digital-twin/pull/384 (cherry pick it or put it in your local changes) so that it stops adding the wrong styles to all the scss types, if you merge terriajs/`mobx` and run your gulp build/watch script, it shouldn't change several dozen files anymore

**Review by soyarsauce — APPROVED** [review]

**Comment by soyarsauce** [comment]
Just https://github.com/TerriaJS/terriajs/pull/4141#discussion_r401322988 and I think good to go, any UI stuff I think we should raise as yet another ticket as a task

File: lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx
Comment (reginapramesti) [reply lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx]: Whoops 😆 these properties became unused ever since I've changed a bunch of things, just forgot to remove them 😬

**Pull request merged**
